### DISCUSSION TITLE
通过Snapcraft安装的CP Editor与通过Github下载.deb文件安装的不一样

### DISCUSSION THREAD

**Discussion by ghost** [cpeditor/cpeditor-cn#9] [提问] [answered]
版本号 [Snapcraft的为最新版](https://snapcraft.io/cpeditor) Github下载的.deb版本为v6.9.3

区别：1.Snapcraft安装的支持[Wakatime](https://github.com/cpeditor/cpeditor/issues/456)，但是通过Github下载的.deb文件安装没有 
![图片](https://user-images.githubusercontent.com/72449367/140534464-6753c994-5e20-41df-952b-6fe13abb2a4a.png)

2.Snapcraft安装在设置扩展时，路径为类似 /doc/104/ex9fegs/clangd 这样的
  而通过Github下载的.deb文件安 为实际路径，如
![图片](https://user-images.githubusercontent.com/72449367/140534795-28f577ee-6f7d-48e5-91b4-8c955b35443e.png)

3.Snapcraft版本似乎无法正确识别clang-format程序

**Comment by ouuan** [comment, marked as answer]
1.  现在 snap 上是 master 分支的版本。暂时没有 stable 版本是因为 snap 是在 v6.9 才有的，按照正常流程是下一个 release 才会有 snap 上的 stable 版本，除非手动进行 release。
2.  程序路径问题好像是 snap 自身的特性，我没用过 snap 不太清楚，好像是 snap 的运行环境和整个系统隔离之类的。

**Reply by ghost** [reply]
所以说，Wakatime这个功能在Github上的版本什么时候会加上呢？
   还有，Snap版本会修复clang-format无法识别的Bug吗？
